File: HWAppFoundation/src/main/java/com/hw/hwdroid/foundation/app/HTitle.java
package com.hw.hwdroid.foundation.app;

import android.support.annotation.StringRes;
import android.view.View;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

/**
 * Created by ChenJ on 2017/2/16.
 */

@Target(ElementType.TYPE)
@Retention(RetentionPolicy.RUNTIME)
public @interface HTitle {
    @StringRes int value() default View.NO_ID;
}

File: HWAppFoundation/src/main/java/com/hw/hwdroid/foundation/app/annotation/HTitle.kt
package com.hw.hwdroid.foundation.app.annotation

import android.support.annotation.StringRes
import android.view.View
import java.lang.annotation.Retention
import java.lang.annotation.RetentionPolicy

/**
 * Theme Light
 * Created by ChenJ on 2017/2/16.
 */
@Retention(RetentionPolicy.RUNTIME)
@Target(AnnotationTarget.CLASS, AnnotationTarget.FILE)
annotation class HTitle(@StringRes val value: Int = View.NO_ID)
